bin/health.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Check that system daemons are running
+
+pid=$(pidof /sbin/rpcbind)
+if [ -z "$pid" ]
+then
+    echo "/sbin/rpcbind not running"
+    exit 1
+fi
+
+pid=$(pidof /sbin/rpc.statd)
+if [ -z "$pid" ]
+then
+    echo "/sbin/rpc.statd not running"
+    exit 1
+fi
+
+pid=$(pidof /usr/sbin/rpc.idmapd)
+if [ -z "$pid" ]
+then
+    echo "/usr/sbin/rpc.idmapd not running"
```

Comment:
Strangely enough, this works with no problems on my local test environment, rpc.idmapd is actually running here:
```
root@5bf58ec1d9c3:/# pidof /usr/sbin/rpc.idmapd
38
root@5bf58ec1d9c3:/# ps aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.0  18864 10432 ?        Ssl  20:48   0:00 /bin/containerbuddy node /opt/nfs/node_modules/sdc-nfs/server.js -f /etc/sdc-nfs.json
root        20  0.0  0.0  37092  2860 ?        Ss   20:48   0:00 /sbin/rpcbind -w
statd       29  0.0  0.3  90352 55940 ?        Ss   20:48   0:00 /sbin/rpc.statd
root        38  0.0  0.0  23368   220 ?        Ss   20:48   0:00 /usr/sbin/rpc.idmapd
root        39  0.0  0.1 973044 30608 ?        Sl   20:48   0:00 node /opt/nfs/node_modules/sdc-nfs/server.js -f /etc/sdc-nfs.json
root      4048  0.1  0.0  20356  3264 ?        Ss   21:28   0:00 /bin/bash
root      4079  0.0  0.0  17512  2084 ?        R+   21:29   0:00 ps aux
root@5bf58ec1d9c3:/# 
```

Odd that it starts up and runs fine locally but does not run on Triton